.github/workflows/ci-actions.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-18.04
     strategy:
       matrix:
-        GOVER: ['1.14', '1.13', '1.12']
+        GOVER: ['1.14', '1.13']
```

Comment:
Why are we removing support for 1.12?

(Latest Debian is still on 1.11.  Ubuntu 19.10 is on 1.12.  Ubuntu 20.04 is on 1.13.)

Reply:
The issue is that crypto/ed25519 appears inside stdlib as of 1.13. 
The public key is an alias of the stdlb crypto/ed25519. This alias plays nicely with type assertions, for example those found in x509.CreateCertificate.
An alternative for not removing support of 1.12 is welcomed.

Reply:
We can control the definition of PublicKey by a build tag.  The "go1.13" build tag is satisfied on Go 1.13 and beyond.

Reply:
The issue with that is that golang.org/x/crypto/ed25519 becomes now a dependency, but only for 1.12, and not for 1.13.
how do we express this conditional dependencies in the go.mod file?

Reply:
I meant: use crypto/ed25519.PublicKey for go1.13 and beyond and simply []byte in previous Go versions.  Go 1.12 and before doesn't support Ed25519 in crypto/x509 anyway.

Reply:
Go support policy is the last two major releases, so I tend to think it's reasonable for libraries to follow the same rules.

https://golang.org/doc/devel/release.html#policy

As far as I am concerned Debian/Ubuntu are too far behind, and library authors should not have to accommodate them. People should be installing Go some other way on those platforms.

Reply:
I'm actually pretty much in favor or removing 1.12, it creates some problems as the API changed a lot.

Debian 10 should have go 1.13: https://tecadmin.net/install-go-on-debian/ for what I understand.

Reply:
The latest in Debian stable (Buster at the moment) is [1.11](https://packages.debian.org/buster/golang).

It's easy to install the latest Go version from source, but that does adds some friction to adoption.

I'm not hell bend on supporting very old version (also as we don't have a big user base yet), but if it's not too much effort, then it does ease adoption.